vendor/bat-native-ads/src/bat/ads/internal/features/ad_rewards/ad_rewards_features.cc

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/features/ad_rewards/ad_rewards_features.h"
+
+#include "base/metrics/field_trial_params.h"
+
+namespace ads {
+namespace features {
+
+namespace {
+
+const char kFeatureName[] = "AdRewards";
+const char kFieldTrialParameterNextPaymentDay[] = "next_payment_day";
+
+const int kDefaultNextPaymentDay = 5;
+
+}  // namespace
+
+const base::Feature kAdRewards{kFeatureName, base::FEATURE_ENABLED_BY_DEFAULT};
+
+bool IskAdRewardsEnabled() {
+  return base::FeatureList::IsEnabled(kkAdRewards);
+}
+
+int GetkAdRewardsNextPaymentDay() {
```

Comment:
typo, should be `GetAdRewardsNextPaymentDay`?

Reply:
Fixed